tests/cases/fourslash/findAllRefsDestructureGetter2.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference path="fourslash.ts" />
+
+// @noLib: true
+
+// @Filename: /a.ts
+////class C {
+////    get [|{| "isWriteAccess": true, "isDefinition": true |}g|](): number { return 0; }
```

Comment:
How is this writeAccess? Its a getter right?

Reply:
It's unintuitive but I think this is correct. `isWriteAccess` tells us when a symbol is given a value; in this case we're giving the getter a value. Just like a `const` declaration can be `isWriteAccess` even though the variable isn't mutable.